helm/alfresco-content-services/values.yaml

```diff
@@ -391,29 +391,123 @@ alfresco-infrastructure:
+  insightEngineImage:
+    repository: quay.io/alfresco/insight-engine
+    tag: "1.4.1"
+    pullPolicy: *searchDockerImagePullPolicy
+    internalPort: *searchDockerImageInternalPort
+  service:
+    name: solr
+    type: ClusterIP
+    externalPort: 80
+  ingress:
+    enabled: false
+    path: /solr
+    # Default solr basic auth user/password: admin / admin
+    # You can create your own with htpasswd utilility & encode it with base640.
+    # Example: `echo -n "$(htpasswd -nbm admin admin)" | base64` # i.e. admin / admin
+    # basicAuth: YWRtaW46JGFwcjEkVVJqb29uS00kSEMuS1EwVkRScFpwSHB2a3JwTDd1Lg==
+    basicAuth:
+    # Comma separated list of IP CIDR to limit search endpoint over the internet
+    whitelist_ips: "0.0.0.0/0"
+  # The parent chart will set the values for "repository.host" and "repository.port"
+  environment:
+    SOLR_CREATE_ALFRESCO_DEFAULTS: alfresco,archive
+    ALFRESCO_SECURE_COMMS: "none"
+    SOLR_OPTS: "-Dsolr.content.dir=/opt/alfresco-search-services/data/contentstore"
+  resources:
+    requests:
+      memory: "250Mi"
+    limits:
+      memory: "1024Mi"
+  # Defines the mounting points for the persistence required by the apps in the cluster
+  # the solr data folder containing the indexes for the alfresco-search-services is mapped to alfresco-content-services/solr-data
+  persistence:
+    enabled: true
+    # existingClaim: "search-master-claim" #Only define if you have a specific claim already created
+    VolumeSizeRequest: 10Gi
+    search:
+      data:
+        mountPath: "/opt/alfresco-search-services/data"
+        subPath: "alfresco-content-services/solr-data"
+    EbsPvConfiguration:
+      fsType: ext4
+  alfresco-insight-zeppelin:
+    enabled: false
+  readinessProbe:
+    initialDelaySeconds: 60
+    periodSeconds: 20
+    timeoutSeconds: 10
+  livenessProbe:
+    initialDelaySeconds: 130
+    periodSeconds: 20
+    timeoutSeconds: 10
+
   repository: &repositoryHostPort
     # The value for "host" is the name of this chart
     host: alfresco-cs
     port: *repositoryExternalPort
   alfresco-insight-zeppelin:
     repository: *repositoryHostPort
-  ingress:
-    # Alfresco Search services endpoint ('/solr') is disabled by default
-    # To enable it please see: acs-deployment configuration table](https://github.com/Alfresco/acs-deployment/tree/master/helm/alfresco-content-services#configuration)
-    enabled: false
+# ingress:
+#    # Alfresco Search services endpoint ('/solr') is disabled by default
+#    # To enable it please see: acs-deployment configuration table](https://github.com/Alfresco/acs-deployment/tree/master/helm/alfresco-content-services#configuration)
+#    enabled: false
+
+  # Apply your secret file in k8s environment to access quay.io images (Example: https://github.com/Alfresco/alfresco-anaxes-shipyard/blob/master/SECRETS.md)
+  # Global definition of Docker registry pull secret which can be overridden from parent ACS Helm chart(s)
+global:
```

Comment:
Just checking the "global" has intentionally been pulled in or was this copy/paste from the search files?

Reply:
Removed duplicate entry